tests/spec/core/data-tests-spec.js

```diff
@@ -63,5 +71,78 @@ describe("Core — data-tests attribute", () => {
         expect(pathname.endsWith(`test${i}.html`)).toBe(true);
       });
     });
+    it(`adds emojis for secure tests`, () => {
+      const li = doc.querySelector("#metadata-tests ul>li:nth-child(1)");
+
+      const fileName = li.children[0].innerText.trim();
```

Comment:
So, having to do "li.children[0]" and "li.children[1]" is ideal here, because it's not clear what element you are working with. Can you please destructure the array into named variables, like (you can probably come up with better names):

```JS
const [fileElement, secureIconElement] = li.children;
```

Similarly for the ones below...

Reply:
typo above, I mean "is not ideal"

Reply:
Yeah Sure 👍